### PR TITLE
Added full solution

### PR DESCRIPTION
**Pull request by RLudchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/MatrixOperation.java [hunk 1]
```diff
@@ -6,18 +6,42 @@
...
+        int[] checkCounterDiagonal = new int[matrix.length];
+        for (int i = 0; i < matrix.length; i++) {
+            checkCounterDiagonal[i] = matrix[i][matrix.length - i - 1];
+        }
+        return checkCounterDiagonal;
     }
 
     public int getMin(int[][] matrix) {
-        return 0;
+        int min = Integer.MAX_VALUE;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MatrixOperation.java]: can you solve without min and max integer values?

Comment (RLudchenko) [reply src/main/java/core/basesyntax/MatrixOperation.java]: done

File: src/main/java/core/basesyntax/MatrixOperation.java [hunk 2]
```diff
@@ -6,18 +6,42 @@
...
+                    min = matrix[i][j];
+                }
+            }
+        }
+        return min;
     }
 
     public int getMax(int[][] matrix) {
-        return 0;
+        int max = matrix[matrix.length - 1][matrix.length - 1];
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MatrixOperation.java]:
```suggestion
        int max = matrix[0][0];
```
you could've start from the first object... why did you begin from the end?

Comment (RLudchenko) [reply src/main/java/core/basesyntax/MatrixOperation.java]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**